docs/reference/upgrading.md

```diff
@@ -9,7 +9,7 @@ The Elastic APM Node.js Agent uses [semantic versioning](https://semver.org/), a
 
 Before upgrading the agent, be sure to review the:
 
-* [Node.js APM Agent release notes](/release-notes/index.md)
+* [Node.js APM Agent release notes](/release-notes/release-notes.md)
```

Comment:
A URL with "release-notes/release-notes" duplicated is odd.

Reply:
I agree. @KOTungseth can we use `/release-notes/index.md`?